CHANGELOG.md

```diff
@@ -38,6 +38,7 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
 - UI: Replaced some Sass variables with CSS variables to allow customization of colors and fonts.
 - Public: Added new enterprise feature `logoCobrand`. When purchased and enabled allows integrator to provide their own logo image to be displayed alongside the Onfido logo.
 - Internal: Use Node 14 LTS for Travis to be consistent with `.nvmrc` and `Dockerfile`.
+- Internal: Enable `strict` mode in tsconfig.json
```

Comment:
Didn't notice this before, but this `CHANGELOG` note feels off based on what's changed in this PR. Also looks like this is the `CHANGELOG` note from #1367 🤔